c302/__init__.py

```diff
@@ -463,62 +442,60 @@ def elem_in_coll_matches_conn(coll, conn):
 
 def _get_cell_info(cells):
 
-    from PyOpenWorm.neuron import Neuron    
-    worm, ctx = get_pyopenworm_worm()
-    if worm is None:
+    try:
+        conn = pyopenworm_connect()
```

Comment:
@mwatts15 This call seems to be slowing down generation of the networks, as it's called once for every cell. Is it possible to cache the connection/context to speed it up?